### PR TITLE
Standardize Join Us on Mobile

### LINKED ISSUE #1607: Standardize Join Us page header in mobile
### Overview
We need to standardize the Join Us page header in mobile to make it consistent with other pages and our design system.

### Action Items
- [x] Increase "Join Us" heading font size to 32px (instead of 24px) **only in mobile**
- [x] Decrease header paragraph text font size to 16px (instead of 18px) **only in mobile**
- [x] Wrap standard ".header-container" class around Join Us header (replacing current ".join-us-banner" class)
   - [x] Include any additional properties (flex-direction etc.) in an additional class called ".header-container--join"
   - [x] See other pages that utilize these classes as an example- [Credits page](https://www.hackforla.org/credits/), etc.
   - [x] Make sure the Join Us page still looks the same in mobile **and** desktop after changing classes

### Resources/Instructions
[Join Us page](https://www.hackforla.org/join)

### PR DESCRIPTION
**Pull request by erikaBell**
1607 

<details>
<summary>Before & After</summary>

Screen Shot 2021-05-23 at 7 46 14 PM

Screen Shot 2021-05-23 at 7 46 21 PM

</details>

### CODE REVIEW

File: _sass/components/_join-us.scss [hunk 1]
```diff
@@ -252,3 +252,24 @@
...
+      font-size: 16px;
+    }
+  }
+
+  .header-container--join {
+    h1{
+      padding-bottom: 0px;
+    }
+    p {
+      text-align: center;
```
Comment (Aveline-art) [inline _sass/components/_join-us.scss:272]: These changes really makes the mobile page pop! Out of curiosity, the original issue (#1607) did not specify to adjust the text-align or padding-bottom properties. Did you decide to add them in or is there a Figma design you are following?

File: pages/join-us.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ---
 
 <div class="join-us-wrapper">
-  <div class="join-us-banner">
+  <div class="join-us-banner header-container--join">
```
Comment (Aveline-art) [inline pages/join-us.html:8]: @daniellex0 The wording from the linked issue (#1607), seems to imply that you want join-us-banner to be replaced. Did you intend for it to be removed or kept as is?

**Comment by erikaBell** [comment]
@Aveline-art this is now closed due to a discussion I had with Danielle earlier today, but I appreciate your review (-:

File: pages/join-us.html (hunk 2 above)
Comment (daniellex0) [reply pages/join-us.html:8]: Thanks @Aveline-art , I updated the language in the original issue to make this more clear :)

**Pull request closed without merging**